test/node/image/image-proc.ts

```diff
@@ -1,64 +1,64 @@
 import { assert, expect } from "chai"
-import Image from "../../src/image/image-proc";
-import {stdCalc} from "../../src/image/utils";
+import Image from "../../../src/image/image-proc";
+import {stdCalc} from "../../../src/image/utils";
```

Comment:
This relative path looks too long, shall we have an alias config for the directory "../../../src" as "@/image/utils"?